From 9e394826072b9d5b752e6ffb34c3d0ca239dd5d6 Mon Sep 17 00:00:00 2001
From: Gleb Natapov <gleb@redhat.com>
Date: Sun, 4 Jul 2010 11:52:47 -0300
Subject: [PATCH 9/9] Fix driftfix option

RH-Author: Gleb Natapov <gleb@redhat.com>
Message-id: <20100704115247.GF4689@redhat.com>
Patchwork-id: 10465
O-Subject: [PATCH RHEL6.0] Fix driftfix option
Bugzilla: 611229
RH-Acked-by: Jes Sorensen <Jes.Sorensen@redhat.com>
RH-Acked-by: Juan Quintela <quintela@redhat.com>
RH-Acked-by: Amit Shah <amit.shah@redhat.com>
RH-Acked-by: Markus Armbruster <armbru@redhat.com>

CONFIG_TARGET_I386 is not set while vl.c is compiled.

BZ: 611229
upstream status: 7e4c0336e2

--
			Gleb.

Signed-off-by: Eduardo Habkost <ehabkost@redhat.com>
---
 vl.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/vl.c b/vl.c
index 64de54c..a3b682d 100644
--- a/vl.c
+++ b/vl.c
@@ -1792,12 +1792,12 @@ static void configure_rtc(QemuOpts *opts)
             exit(1);
         }
     }
-#ifdef CONFIG_TARGET_I386
+#ifdef TARGET_I386
     value = qemu_opt_get(opts, "driftfix");
     if (value) {
-        if (!strcmp(buf, "slew")) {
+        if (!strcmp(value, "slew")) {
             rtc_td_hack = 1;
-        } else if (!strcmp(buf, "none")) {
+        } else if (!strcmp(value, "none")) {
             rtc_td_hack = 0;
         } else {
             fprintf(stderr, "qemu: invalid option value '%s'\n", value);
-- 
1.7.0.3

